templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
Don't worry about that guys, it's only a misunderstand. 
Please review the package carefully and if you like it and you want to include zookeeper features in your package, i'm happy to make a new PR to rancher-catalog zookeeper, it's no problem..Or may be i think about that funny name to be in here... ;P

@vincent99 i understand your point, and almost i share your opinion, but i think that in this particular case, to have a particular instance of zookeeper is more than justified due to the kafka nature. In terms of scale and/or performance, to have a shared zookeeper may be a problem for kafka, and eventually for others apps that try to use the same shared zookeeper.